manifests/server.pp

```diff
@@ -334,6 +337,7 @@
   Array[String] $cipher_suites = $::puppet::server_cipher_suites,
   Optional[String] $config_version = $::puppet::server_config_version,
   Integer[0] $connect_timeout = $::puppet::server_connect_timeout,
+  Integer[0] $server_web_idle_timeout = $puppet::params::server_web_idle_timeout,
```

Comment:
in the template, it would also be just `@web_idle_timeout` then, as the var is locally available